src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/IPOContextModelSnapshot.cs

```diff
@@ -239,8 +239,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<Guid>("Guid")
                         .HasColumnType("uniqueidentifier");
 
-                    b.Property<int?>("InvitationId")

```

Comment:
ohh .. I guess this change has been added since you during the process had the InvitationId as MAPPED property 
We should compare our schema with my schema (since I've not run the migration yet)

I'm pretty sure the solution is to revert this change, but we must be sure .. need to talk about it

Reply:
Right. So what do we need to compare here? An export of the loca data model?